mne/io/curry/tests/test_curry.py

```diff
@@ -288,6 +303,21 @@ def test_read_raw_curry_rfDC(fname, tol, mock_dev_head_t, tmp_path):
     assert (angles < 20).sum() > 100
 
 
+@testing.requires_testing_data
+@pytest.mark.parametrize(
+    "fname",
+    [
+        pytest.param(curry7_bdf_file, id="curry 7"),
+        pytest.param(curry8_bdf_file, id="curry 8"),
+    ],
+)
+def test_read_raw_curry_preload_equal(fname):
+    """Test raw identity with preload=True/False."""
+    raw1 = read_raw_curry(fname, preload=False)
+    raw1.load_data()
+    assert raw1 == read_raw_curry(fname, preload=True)
```

Comment:
if you use just `_test_raw_reader` it should automatically check preloading equivalence (that func has a `test_preloading=True` default)